Pipfile

```diff
@@ -0,0 +1,11 @@
+[[source]]
```

Comment:
Not clear why this was picked up.

Reply:
To me it seems that the two Pipfiles were created and added by mistake and can be removed from the PR

Reply:
I did not intentionally create these. They were created when I followed the venv installation instructions and ran:

`pipenv install --dev`

I think that I may have figured out what happened. I was in the vehicle_signal_specification directory instead of vss-tools when I ran the command.

Nonetheless, I don't see Pipfile or Pipfil.lock listed in either .gitignore files.

Reply:
I have another commit to remove these files.

Reply:
It is indeed a bit unclear in our VSS README where and how to setup the pyenv, it just refers to vss-tools and then it is easy to try to run pyenv from the wrong directory. We could add Pipfile* to .gitignore if we find it useful, our current approach to .gitignore is quite permissive, if anyone wants something to be added because for example their IDE add some files then we accept it.

Reply:
I have made a note to add some clarifications.

Now, if I could only sort out how best to do proper squashes, LOL. (The rebase approach I normally use seems to run into issues.)